src/darwin/Framework/CHIP/MTRThreadOperationalDataset.h

```diff
@@ -80,18 +80,18 @@ extern size_t const MTRSizeThreadPANID MTR_NEWLY_AVAILABLE;
  *              returned.  In particular, it's expected to be a 16-bit unsigned
  *              integer stored as 2 bytes in host order.
  */
-- (nullable instancetype)initWithNetworkName:(NSString *)networkName
-                               extendedPANID:(NSData *)extendedPANID
-                                   masterKey:(NSData *)masterKey
-                                        PSKc:(NSData *)PSKc
-                               channelNumber:(NSNumber *)channelNumber
-                                       panID:(NSData *)panID MTR_NEWLY_AVAILABLE;
+- (instancetype _Nullable)initWithNetworkName:(NSString *)networkName
+                                extendedPANID:(NSData *)extendedPANID
+                                    masterKey:(NSData *)masterKey
+                                         PSKc:(NSData *)PSKc
+                                channelNumber:(NSNumber *)channelNumber
+                                        panID:(NSData *)panID MTR_NEWLY_AVAILABLE;
```

Comment:
This should be `PANID` if we want to follow the convention (and technically `PKSc` should probably be `PKSC` as well)

Reply:
So I poked around and https://developer.apple.com/documentation/threadnetwork/thcredentials?changes=_2_6&language=objc uses `panID` and `PKSC` respectively...  Maybe we should just match that?

Reply:
sounds good to me

Reply:
I filed https://github.com/project-chip/connectedhomeip/issues/23484 for the moment and will look into it once the other stuff is relanded.